compiler/modules/nback/build.bal

```diff
@@ -1343,6 +1343,19 @@ function buildEqualFloat(llvm:Builder builder, Scaffold scaffold, boolean exact,
     return buildStoreBoolean(builder, scaffold, b, reg);
 }
 
+function generateStringEquality(llvm:Builder builder, Scaffold scaffold, CmpEqOp op, bir:EqualityInsn insn, llvm:Value lhsValue, llvm:Value rhsValue, bir:Register result) {
+    foreach var operand in insn.operands {
+        if !(operand is string) {
```

Comment:
Do not use `continue` so you can avoid the cast later on.

Reply:
Fixed